SeaPublicWebsite/Services/EnergyEfficiency/PdfGeneration/PdfGenerationService.cs

```diff
@@ -32,6 +33,11 @@ public async Task<Stream> GeneratePdf(string path)
         var page = await browser.NewPageAsync();
         await page.AuthenticateAsync(new Credentials
             { Username = basicAuthConfiguration.Username, Password = basicAuthConfiguration.Password });
+        await page.SetCookieAsync(new CookieParam { 
+            Name = "service_language", 
+            Value = $"c={CultureInfo.CurrentCulture.Name}|uic={CultureInfo.CurrentUICulture.Name}",
```

Comment:
This cookie value looks like it's correct, but why not use `CookieRequestCultureProvider.MakeCookieValue()` like we do in `EnergyEfficiencyController()`?

Reply:
Good point, updated this!